tensorflow/compiler/tf2tensorrt/common/utils.h

```diff
@@ -32,9 +32,41 @@ std::tuple<int, int, int> GetLoadedTensorRTVersion();
 
 #if GOOGLE_CUDA && GOOGLE_TENSORRT
 
+#include "tensorflow/core/platform/errors.h"
 #include "tensorflow/core/platform/logging.h"
+#include "tensorflow/core/platform/status.h"
 #include "third_party/tensorrt/NvInfer.h"
 
+// Use this macro within functions which return a Status or StatusOR<T> to check
+// boolean conditions. If the condition fails, it returns an
+// errors::Internal message with the file and line number.
+#define TRT_ENSURE(x)                                                        \
+  if (!(x)) {                                                                \
+    return errors::Internal(__FILE__, ":", __LINE__, " TRT_ENSURE failure"); \
+  }
+
+// Check that a Status or StatusOr<T> object does not carry an error message.
+// For pointer-containing StatusOr<T*>, use the below TRT_ENSURE_PTR_OK macro.
```

Comment:
Let's move this line to the end of the comment block. Otherwise, the meaning of "it" in the the following sentence is not clear.